attributeerror nonetype object has no attribute 'splitlines'

Log http://paste.ubuntu.com/25929062/

```
2017-11-09 22:12:55,198 [INFO] conjure-up/kubernetes-core - common.py:52 - Waiting for deployment to settle.
2017-11-09 22:22:55,498 [DEBUG] conjure-up/kubernetes-core - events.py:53 - Setting ModelSettled at conjureup/controllers/deploy/common.py:57 in task _wait_for_applications at conjureup/controllers/deploy/tui.py:13
2017-11-09 22:22:55,498 [INFO] conjure-up/kubernetes-core - common.py:59 - Model settled.
2017-11-09 22:22:55,504 [INFO] conjure-up/kubernetes-core - step.py:196 - Running step: 02_get-kubectl after-deploy.
2017-11-09 22:22:55,633 [DEBUG] conjure-up/kubernetes-core - step.py:233 - Storing environment
2017-11-09 22:22:55,636 [DEBUG] conjure-up/kubernetes-core - step.py:239 - Executing script: /home/ubuntu/.cache/conjure-up/kubernetes-core/steps/02_get-kubectl/after-deploy
2017-11-09 22:23:02,733 [INFO] conjure-up/kubernetes-core - step.py:196 - Running step: 03_cluster-info after-deploy.
2017-11-09 22:23:02,854 [DEBUG] conjure-up/kubernetes-core - step.py:233 - Storing environment
2017-11-09 22:23:02,857 [DEBUG] conjure-up/kubernetes-core - step.py:239 - Executing script: /home/ubuntu/.cache/conjure-up/kubernetes-core/steps/03_cluster-info/after-deploy
2017-11-09 22:23:04,139 [DEBUG] conjure-up/kubernetes-core - events.py:53 - Setting Error at conjureup/events.py:148
2017-11-09 22:23:04,140 [ERROR] conjure-up/kubernetes-core - events.py:160 - Unhandled exception in <Task finished coro=<RunStepsController.run_steps() done, defined at /snap/conjure-up/812/lib/python3.6/site-packages/conjureup/controllers/runsteps/tui.py:17> exception=AttributeError("'NoneType' object has no attribute 'splitlines'",)>
Traceback (most recent call last):
  File "/snap/conjure-up/812/lib/python3.6/site-packages/conjureup/controllers/runsteps/tui.py", line 25, in run_steps
    step.result = await step.after_deploy(utils.info)
  File "/snap/conjure-up/812/lib/python3.6/site-packages/conjureup/models/step.py", line 110, in after_deploy
    return await self.run(PHASES.AFTER_DEPLOY, msg_cb)
  File "/snap/conjure-up/812/lib/python3.6/site-packages/conjureup/models/step.py", line 260, in run
    for line in err_log.splitlines():
AttributeError: 'NoneType' object has no attribute 'splitlines'
2017-11-09 22:23:04,148 [DEBUG] conjure-up/kubernetes-core - events.py:53 - Setting Shutdown at conjureup/events.py:165
2017-11-09 22:23:04,149 [DEBUG] conjure-up/kubernetes-core - events.py:53 - Received Shutdown at conjureup/events.py:175
2017-11-09 22:23:04,149 [INFO] conjure-up/kubernetes-core - events.py:179 - Shutting down
2017-11-09 22:23:04,149 [INFO] conjure-up/kubernetes-core - app_config.py:190 - Storing conjure-up state
2017-11-09 22:23:04,182 [INFO] conjure-up/kubernetes-core - app_config.py:194 - State saved in model config
2017-11-09 22:23:04,182 [INFO] conjure-up/kubernetes-core - events.py:189 - Disconnecting model
2017-11-09 22:23:04,185 [INFO] conjure-up/kubernetes-core - events.py:191 - Disconnected
```

This is with kubernetes-core on localhost using latest --edge

fyi @johnsca

--- a/conjureup/utils.py
+++ b/conjureup/utils.py
@@ -194,8 +194,8 @@ async def arun(cmd, input=None, check=False, env=None, encoding='utf8',
         if errf:
             await errf.close()
 
-    stdout_data = ''.join(data.get('stdout', [])) or None
-    stderr_data = ''.join(data.get('stderr', [])) or None
+    stdout_data = ''.join(data.get('stdout', [])) if proc.stdout else None
+    stderr_data = ''.join(data.get('stderr', [])) if proc.stderr else None
 
     if check and proc.returncode != 0:
         raise subprocess.CalledProcessError(proc.returncode,

